job.go

```diff
@@ -122,3 +122,18 @@ func (j *Job) LimitRunsTo(n int) {
 func (j *Job) shouldRun() bool {
 	return !j.runConfig.finiteRuns || j.runCount < j.runConfig.maxRuns
 }
+
+// LastRun returns the time the job was run last
+func (j *Job) LastRun() time.Time {
+	return j.lastRun
```

Comment:
we probably want to lock the mutex here so we don't get a data race if another go routine is currently modifying the job when this is called. You can check the tests with `go test -race github.com/go-co-op/gocron`

Reply:
This might be a dumb question. But I just saw that we are locking mutex while modifying the `lastRun` value. Do we also need to lock when we read it?

Reply:
This is not showing up in `go test -race`.

Reply:
yes - otherwise you couldn't guarantee the value isn't changing at the moment you read it. This has some good examples https://golangbot.com/mutex/

Reply:
Thanks!